src/features/shujinosuke.js

```diff
@@ -266,6 +289,62 @@ ${JSON.stringify(Object.fromEntries(global_state), null, 2)}
     }
   );
 
+  controller.hears(
+    /^(誰いる？||今いる人は？)$/,
+    "direct_mention",
+    async (bot, message) => {
+      await bot.changeContext(message.reference);
+      controller.trigger("reply_attendees", bot, message);
+    }
+  );
+
+  controller.on("reply_attendees", async (bot, message) => {
+    const message_txt = `
+  :male-technologist: こちらが現在Slack上でアクティブな方々です！
+  :speaker: DMを送るには名前をクリックし、「メッセージ」を選択してください！
+  :warning: あくまでSlackの状態がアクティブなユーザーを表示しているので、勤務しているとは限りません。
+  `;
+    const all_members_response = await bot.api.users.list({});
+    const all_members = all_members_response.members;
+    // remove deleted accounts, bots and outsourcees
+    const reduced_members = all_members.filter(
+      (member) => !member.deleted && !member.is_bot && !member.is_restricted
+    );
+    const attendees_id = await Promise.all(
+      reduced_members.map(async (member) => {
+        const presence_response = await bot.api.users.getPresence({
+          user: member.id,
+        });
+        if (presence_response.presence === "active") {
+          return member.id;
+        }
```

Comment:
ここを
```suggestion
        if (presence_response.presence === "active") {
          return member.id;
        } else {
          return null
        }
```
明示的にこうしましょう。するとその後のfilterがnullを弾いてんだなとわかる。